docs/README.md

```diff
@@ -8,7 +8,7 @@ Sphinx is used for documentation. You can generate HTML locally with the
 following:
 
 ```
-$ pip install -r requirements-docs.txt
+$ pip install -r requirements-docs-lock.txt
```

Comment:
We can probably leave this change out to not overly constrain users when building the docs.